<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MVCAMFullSettingsWidget</class>
 <widget class="QWidget" name="MVCAMFullSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MVCAMFullSettingsWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_15">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,4">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>分辨率</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QComboBox" name="res_combobox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>查表映射变换</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>伽马</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="gamma_horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_gamma"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>饱和度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="contrast_horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_contrast"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_16">
       <property name="title">
        <string>曝光设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="exposure_mode_auto">
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="exposure_mode_manual">
            <property name="text">
             <string>手动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="exposure_manual">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>增益</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_gain">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_gain"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>曝光时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_exposure_time">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_exposure_time"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="exposure_auto">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QCheckBox" name="flicker_checkBox">
               <property name="text">
                <string>抗频闪</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_50HZ">
               <property name="text">
                <string>50HZ</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_60HZ">
               <property name="text">
                <string>60HZ</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>亮度目标</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="AE_horizontalSlider">
               <property name="maximum">
                <number>300</number>
               </property>
               <property name="value">
                <number>99</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="AE_spinBox">
               <property name="maximum">
                <number>300</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,4,2,0">
     <item>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>采集模式</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QRadioButton" name="radioButton_trigger_hardware">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>硬触发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_software_trigger">
          <property name="text">
           <string>软触发</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_collect">
          <property name="text">
           <string>连续采集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="software_trigger_once_button">
          <property name="text">
           <string>软件触发一次</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_wb">
       <property name="title">
        <string>ISP颜色处理</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="AWB_once_button">
          <property name="text">
           <string>一键白平衡</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>红色增益</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_gain_r">
            <property name="maximum">
             <number>400</number>
            </property>
            <property name="pageStep">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_gain_r">
            <property name="maximum">
             <number>400</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>绿色增益</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_gain_g">
            <property name="maximum">
             <number>400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_gain_g">
            <property name="maximum">
             <number>400</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>蓝色增益</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_gain_b">
            <property name="maximum">
             <number>400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_gain_b">
            <property name="maximum">
             <number>400</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>饱和度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_saturation">
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_saturation">
            <property name="maximum">
             <number>200</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>ISP图像处理</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QCheckBox" name="checkBox_isp_h">
            <property name="text">
             <string>水平镜像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_isp_v">
            <property name="text">
             <string>垂直镜像</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>锐度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_isp_acutance">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_isp_acutance"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_speed">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>速度模式</string>
       </property>
       <widget class="QFrame" name="frame_speed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>310</width>
          <height>35</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>-20</x>
          <y>110</y>
          <width>652</width>
          <height>78</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QRadioButton" name="radioButton_B">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>12</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_A">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>12</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_C">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_para_save">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>10</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_para_acquiesce">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>70</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>默认</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_para_load">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>140</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>从文件中加载</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_D">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>42</y>
           <width>40</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>645</width>
          <height>34</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>该相机是黑白相机，某些参数无法调节</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>模式</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_AWB_auto">
             <property name="text">
              <string>自动</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_13">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>手动</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="flashlight">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>20</y>
          <width>122</width>
          <height>23</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>131</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>闪光灯控制</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QFrame" name="flashlight_polarity">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>110</width>
              <height>30</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>110</width>
              <height>30</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QRadioButton" name="flashlight_h">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>5</y>
               <width>40</width>
               <height>22</height>
              </rect>
             </property>
             <property name="text">
              <string>高</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="flashlight_l">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>5</y>
               <width>50</width>
               <height>22</height>
              </rect>
             </property>
             <property name="text">
              <string>低</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_26">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>25</y>
              <width>62</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>有效极性</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="flashlight_auto">
            <property name="geometry">
             <rect>
              <x>5</x>
              <y>5</y>
              <width>55</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>自动</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="flashlight_manual">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>5</y>
              <width>55</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>手动</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
